rs/nns/inspector/src/main.rs

```diff
@@ -279,69 +271,3 @@ fn decode_gtc_stable_memory(gtc_pb: PathBuf, output: &Path, rs: &Path) {
     }
     eprintln!("Wrote gtc_accounts_EXTRACT.csv");
 }
-
-/// The type of record expected to be found in the ledger canister's stable
-/// memory.
-#[derive(serde::Serialize)]
-struct LedgerBalanceRecord {
-    account_identitifier: String,
-    balance_e8s: u64,
-}
-
-/// Decode stable memory for the ledger canister.
-fn decode_ledger_stable_memory(cbor: PathBuf, output: &Path) {
-    // For the same argument as above, do NOT deserialize to a specific struct.
-    // This is an audit tool, we don't want any risk of dropping data.
-    // So go the schema-less way.
-    use serde_cbor::value::Value;
-    let value_res: serde_cbor::Result<Value> =
-        serde_cbor::from_reader(File::open(cbor.clone()).unwrap());
-    let val = match value_res {
-        Err(e) => {
-            eprintln!(
-                "Could not parse the cbor for the ledger stable memory: {}",
-                e
-            );
-            return;
-        }
-        Ok(v) => v,
-    };
-    // The following is very slow and the output is very verbose. Serialization to
-    // json is not a viable alternative as some keys are not strings, which is
-    // allowed in cbor but not in json.
-    match write!(
-        File::create(output.join("ledger_stable_memory.txt")).unwrap(),
-        "{:#?}",
-        val
-    ) {
-        Ok(()) => eprintln!("Wrote ledger_stable_memory.txt"),
-        Err(e) => eprintln!("Could not write ledger_stable_memory.txt: {}", e),
-    };
-
-    let ledger: ledger_canister::Ledger = match serde_cbor::from_reader(File::open(cbor).unwrap()) {
```

Comment:
I guess this tool is not used very often, as this would fail since https://dashboard.internetcomputer.org/proposal/133946 was executed. The ledger stable memory now contains the stable structures memory manager and cannot be directly deserialized to the Ledger struct. Also, the balances are no longer stored in the Ledger struct directly but in BALANCES_MEMORY instead. The reason I am removing it only now is that the `balances().store` below no longer provides an `iter()`.  I suggest we remove this code now and reimplement it if needed later.